app/controllers/foreman_ansible/concerns/hostgroups_controller_extensions.rb

```diff
@@ -0,0 +1,29 @@
+module ForemanAnsible
+  module Concerns
+    # Extra methods to enforce Ansible roles on a host or multiple hosts
+    module HostgroupsControllerExtensions
+      extend ActiveSupport::Concern
+      include ForemanTasks::Triggers
+
+      def play_roles
```

Comment:
By the regular GET action I mean /hostgroups/play_roles - it should be /api/v2/hostgroups/:id/play_roles